middleware/middleware_ratelimit.go

```diff
@@ -30,47 +28,43 @@ const (
 	DefaultServiceRate = 0
 )
 
+// RateLimitConfig is the configuration for the rate limiter middleware.
 type RateLimitConfig struct {
-	Enabled     bool `toml:"enabled"`
-	PublicRate  int  `toml:"public_requests_per_minute"`
-	AccountRate int  `toml:"user_requests_per_minute"`
-	ServiceRate int  `toml:"service_requests_per_minute"`
+	// Enabled turns on/off the middleware.
+	Enabled bool `toml:"enabled"`
+	// PublicRPM is the rate limit for Public sessions expressed as number of requests per minute.
```

Comment:
I think you forgot to add `BaseRequestCost` config param..?

Reply:
BaseRequestCost has been moved to Options:

https://github.com/0xsequence/quotacontrol/blob/96e378ebb5fcb96a471598751eca7dc158547db6/middleware/common.go#L18-L22

Since it was used in two places:

https://github.com/0xsequence/quotacontrol/blob/96e378ebb5fcb96a471598751eca7dc158547db6/middleware/middleware_cost.go#L18-L24

https://github.com/0xsequence/quotacontrol/blob/96e378ebb5fcb96a471598751eca7dc158547db6/middleware/middleware_ratelimit.go#L105-L116

So it is was easy to pass it to both middelware in a clean way, plus we can set the default as we do for other things in here.